modules/@apostrophecms/ui/ui/apos/scss/global/_tooltips.scss

```diff
@@ -0,0 +1,86 @@
+// Styles based on the v-tooltip example styles:
+// https://github.com/Akryum/v-tooltip/blob/83615e394c96ca491a4df04b892ae87e833beb97/demo-src/src/App.vue#L179-L303
+.apos-tooltip {
+  z-index: $z-index-notifications;
+  display: block;
+
+  .apos-tooltip__inner {
+    @include type-large;
```

Comment:
use `type-small`

Reply:
That looked pretty small versus the design, but I can do that.